specification/loadtestservice/resource-manager/Microsoft.LoadTestService/loadtesting/readme.md

```diff
@@ -0,0 +1,114 @@
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.LoadTestService/loadTests/{loadTestName}/limits/maxMonthlyVirtualUserHours/set"]
+    suppress: ResourceNameRestriction
+    reason: The parent loadTests resource (existing API) does not have pattern restriction.
+  - where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.LoadTestService/loadTests/{loadTestName}/limits/maxMonthlyVirtualUserHours/set"]
+    suppress: PathForResourceAction
+    reason: API design for post action for the load testing resource limits.
+suppressions:
+  - code: ResourceNameRestriction
+    from: loadtestservice.json
+    reason: Existing API, will be a breaking change for this api-version.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.LoadTestService/locations/{location}/quotas/{quotaBucketName}"]
+      - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.LoadTestService/locations/{location}/quotas/{quotaBucketName}/checkAvailability"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.LoadTestService/loadTests/{loadTestName}"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.LoadTestService/loadTests/{loadTestName}/outboundNetworkDependenciesEndpoints"]
+  - code: RequiredPropertiesMissingInResourceModel
+    from: loadtestservice.json
+    reason: Existing API, will be a breaking change for this api-version.
+    where:
+      - $.definitions["PagedOutboundEnvironmentEndpoint"]
+```
+
+### Tag: package-2023-12-01-preview
+
+These settings apply only when `--tag=package-2023-12-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-12-01-preview'
+input-file:
+  - preview/2023-12-01-preview/loadtestservice.json
+suppressions:
+  - code: ResourceNameRestriction
+    from: loadtestservice.json
+    reason: Existing API, will be a breaking change for this api-version.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.LoadTestService/locations/{location}/quotas/{quotaBucketName}"]
+      - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.LoadTestService/locations/{location}/quotas/{quotaBucketName}/checkAvailability"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.LoadTestService/loadTests/{loadTestName}"]
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.LoadTestService/loadTests/{loadTestName}/outboundNetworkDependenciesEndpoints"]
+  - code: RequiredPropertiesMissingInResourceModel
+    from: loadtestservice.json
+    reason: Existing API, will be a breaking change for this api-version.
```

Comment:
are you planning another api version to fix it?

Reply:
Not yet, this is led by another service team so can't quote right away.